service/src/main/java/com/kodemon/service/interfaces/TrainerFightService.java

```diff
@@ -22,9 +22,10 @@
      * This method also takes care of giving new badge to the challenger if he wins
```

Comment:
Why are these changes committed here? They are also in the #90. This creates the mess a bit. I don't think that something gets wrong in merging, but it shouldn't be here.